### ISSUE TITLE
Crash After breaking premade circuit with tinkers pick ?

### ISSUE THREAD

**Issue by Bennyboy1695** [amadornes/SCM-Localization-Issues#309]
Placed a circuit on a double slab block, then put the pre-made schematic on it and attempted to break it then got this crash - http://pastebin.com/B4padaiY

**Comment by Milii20** [comment]
i literally just got the same crash, same nullPointerException at com.amadornes.rscircuits.part.PartCircuit.getPickBlock(PartCircuit.java:952)

(with tinkers pickaxe too)

**Comment by rgtexa** [comment]
Same crash, but with a custom circuit (a simple NOT gate). Breaking circuit with Tinkers' pickaxe caused the game to crash.

https://gist.github.com/rgtexa/10a15ba9e1c0b8922b689be25479bed1

Running SkyFactory 3 v3.0.6
Minecraft 1.10.2
Forge 2215
SCM 1.0.4

**Comment by desht** [comment]
I've seen this too, but it doesn't just happen with Tinker's tools.  I broke a circuit containing an inventory scanner, subtractor & constant generator; first with an EnderIO dark pick, and then with a bare hand - crashed in both cases, with the same stack trace as in the pastebin above.  Skyfactory 3.0.5 & SCM 1.0.4.

**Comment by dshadowwolf** [comment]
Yes, brand new world, in creative put in an AND-gate, a pair of redstone torches for two inputs and a WR-CBE receiver for the third - everything worked. Switch to survival, try to pick up using a diamond pick and get a crash - pretty much the same as reported above.

Oddly, removing WAILA/HWYLA stops this, so there might be a mod-interaction bug involved. Contacting TehNut.

**Comment by TehNut** [comment]
This crash is deep in SCM code. There's nothing I can do about it on my end.
